devops/arangodb.yml

```diff
@@ -42,11 +47,39 @@
       shell: ARANGODB_DEFAULT_ROOT_PASSWORD={{ arangodb_root_password }} arango-secure-installation
       become: true
 
+    - name: Copy the compressed ssl certificate files to the server
+      copy:
+        src: '{{ ssl_cert_path }}'
+        dest: /home/ubuntu/
+        follow: yes
+        mode: u=rw,g=,o=
+
+    - name: Combine the chain and the private key into one file (required by arango)
+      shell:
+        chdir: /home/ubuntu/
+        cmd: cat fullchain.pem privkey.pem > server.pem
+
+    - name: Set the combined file as 600 permission
+      file:
+        path: /home/ubuntu/server.pem
+        mode: u=rw,g=,o=
+
+    - name: Add an exception to the permissions for the arangodb user
+      shell:
+        chdir: /home/ubuntu/
+        cmd: setfacl -m u:arangodb:r /home/ubuntu/server.pem
+
+    - name: Add the ssl filepath to the config
+      lineinfile:
+        path: /etc/arangodb3/arangod.conf
+        regexp: '^\[ssl\]'
+        line: "\n[ssl]\nkeyfile = /home/ubuntu/server.pem"
```

Comment:
I'm pretty sure this won't be idempotent--I imagine that every time the playbook runs, this replacement will occur, resulting in multiple "[ssl]" blocks with repeated "keyfile" lines.

Can you confirm this?

Reply:
After running it twice, we're left with this:

```
[ssl]
keyfile = /home/ubuntu/server.pem
keyfile = /home/ubuntu/server.pem
```

I'll update the regex

Reply:
I had to switch to `blockinfile`. Fixed in cfed808